$NetBSD: patch-aa,v 1.3 2011/11/09 18:38:02 drochner Exp $

deal with the fact that ENOTSUPP and EOPNOTSUPP is identical on Linux but not in general

--- lib/backupclient/BackupClientFileAttributes.cpp.orig	2011-01-08 21:35:23.000000000 +0000
+++ lib/backupclient/BackupClientFileAttributes.cpp
@@ -661,7 +661,7 @@ void BackupClientFileAttributes::FillExt
 		}
 		else if(listSize<0)
 		{
-			if(errno == EOPNOTSUPP || errno == EACCES)
+			if(errno == EOPNOTSUPP || errno == EACCES || errno == ENOTSUP)
 			{
 				// fail silently
 			}
@@ -1016,7 +1016,7 @@ void BackupClientFileAttributes::WriteEx
 		xattrOffset += sizeof(u_int32_t);
 
 		// FIXME: Warn on EOPNOTSUPP
-		if(::lsetxattr(Filename, key, buffer+xattrOffset, valueSize, 0)!=0 && errno!=EOPNOTSUPP)
+		if(::lsetxattr(Filename, key, buffer+xattrOffset, valueSize, 0)!=0 && errno!=EOPNOTSUPP && errno!=ENOTSUP)
 		{
 			BOX_LOG_SYS_ERROR("Failed to set extended attributes "
 				"on file '" << Filename << "'");
